pkg/validate/dynamic/loadbalancerprofile_test.go

```diff
@@ -8,13 +8,14 @@ import (
 	"strconv"
 	"testing"
 
-	mgmtnetwork "github.com/Azure/azure-sdk-for-go/services/network/mgmt/2020-08-01/network"
+	sdknetwork "github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork/v2"
 	"github.com/Azure/go-autorest/autorest/to"
 	"github.com/sirupsen/logrus"
 	"go.uber.org/mock/gomock"
+	"k8s.io/utils/ptr"
```

Comment:
Why are we using this lib instead of just going with `to.Ptr`?

Reply:
Because it's already deprecated.
https://github.com/Azure/go-autorest

Reply:
The `to` functionality moved into the core sdk: `github.com/Azure/azure-sdk-for-go/sdk/azcore/to`

Reply:
There are already usages of `ptr.To` in other places. We should discuss which should be used, but anyway we need to change the code from `to.Ptr*` to `to.Ptr` or `ptr.To`.
I don't have any preference, but I don't think it's a blocker for this PR.

Reply:
https://redhat-internal.slack.com/archives/C02ULBRS68M/p1731940533915429